### PR TITLE
Add example configuration using url instead of multiple parameters

### PR DESCRIPTION
**Pull request by samvdb**

### CODE REVIEW

**Comment by samvdb** [comment]
Can we merge this?

File: README.md [hunk 1]
```diff
@@ -71,6 +71,37 @@ doctrine:
...
+#### Warning:
+The url **MUST** be schemaless. An `InvalidArgumentException` will be thrown otherwise.
+```
+Facile\DoctrineMySQLComeBack\Doctrine\DBAL\Connection needs a driver that implements ServerGoneAwayExceptionsAwareInterface 
+```
+
+```yaml
+# Doctrine example Configuration
+parameters:
+    # Adds a fallback DATABASE_URL if the env var is not set.
```
Comment (Jean85) [inline README.md]: This is wrong. If the cache warmup triggers a DB connection in Doctrine it's not this lib's fault, but it's due to the fact that you didn't specify a DB version/charset in the connection, so Doctrine is forced to connect to it to obtain those.

**Comment by antondemanoff** [comment]
Usage example with url is very needed in the readme. I've spent a lot of time finding the root of problem, which is in scheme in DATABASE_URL (https://github.com/doctrine/dbal/issues/3209)

File: README.md (hunk 1 above)
Comment (peelandsee) [reply README.md]: @samvdb 
may you change the PR accordingly with @Jean85 suggestion?
should be really fast, then we'll merge this.
thanks,

Comment (peelandsee) [reply README.md]: @samvdb any update on this?

**Comment by peelandsee** [comment]
no answer from @samvdb,
closing this now

**Pull request closed without merging**